dotnet/samples/Demos/AotCompatibility.TestApp/Tests/PromptEchoChatCompletionService.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Runtime.CompilerServices;
+using Microsoft.SemanticKernel;
+using Microsoft.SemanticKernel.ChatCompletion;
+
+namespace AotCompatibility.TestApp.Tests;
+internal sealed class PromptEchoChatCompletionService : IChatCompletionService
```

Comment:
Just curious why are we introducing this complexity to the demonstration, some of samples needs to use `Services.AddSingleton<IChatCOmpletionService>` instantiating this service which is not a common path of usage.

If this project is being used for Testing Purposes (Not demonstration) I suggest leaving those tests in a separate project.

```suggestion
internal sealed class PromptEchoChatCompletionService : IChatCompletionService
```

Reply:
Fixed.